codespell_lib/data/dictionary.txt

```diff
@@ -43081,6 +43093,7 @@ preshusly->preciously
 presice->precise, preside,
 presicely->precisely
 presicion->precision
+presidence->precedence (maybe for rare)
```

Comment:
Yes, I would say we move it to _rare_. Although I cannot find it in [SCOWL (and Friends)](http://app.aspell.net/lookup?dict=en_US-large;words=presidence), it is part of [OED](https://www.oed.com/search/dictionary/?q=presidence) and some other curated dictionaries.

For the sake of French native speakers, I would also recommend:
```suggestion
presidence->precedence, presidency,
```

Comparison of `presidence` vs. `presidency` usage:
https://books.google.com/ngrams/graph?content=presidence%2Cpresidency&year_start=1800&year_end=2019&corpus=en-2019